src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
+/* eslint-disable no-console */
 'use strict';
 
-// write your code here
+const parsed = [
+  ...document
+    .body
+    .querySelectorAll('.population')]
+  .map(el =>
+    Number(
+      el
+        .innerText
+        .split(',')
+        .join('')));
+
+const total = parsed
+  .reduce((prev, curr) =>
+    prev + curr
+  , 0);
```

Comment:
```suggestion
const total = parsed.reduce((prev, curr) => prev + curr, 0);
```
Кажется в одну строку влезает

Reply:
Ага